google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/data/v2/BigtableDataClientFactoryTest.java

```diff
@@ -169,10 +170,17 @@ public void tearDown() {
 
   @Test
   public void testNewClientsShareTransportChannel() throws Exception {
-
     // Create 3 lightweight clients
 
-    try (BigtableDataClientFactory factory = BigtableDataClientFactory.create(defaultSettings);
+    // Builtin metrics will call getCredentialsProvider at which point it'll be a
+    // FixedCredentialProvider.
+    // So disabling in the test code it's fine.
```

Comment:
I dont understand what this means

Reply:
In EnhancedBigtbaleStub pass in the ClientContext so we get get Credentials from client context

Reply:
Will update the test after refactoring PR is merged:  https://github.com/googleapis/java-bigtable/pull/2102

Reply:
I think you can do this now?